tensorboard/plugins/histogram/summary.py

```diff
@@ -16,13 +16,20 @@
 
 A histogram summary stores a list of buckets. Each bucket is encoded as
 a triple `[left_edge, right_edge, count]`. Thus, a full histogram is
-encoded as a tensor of dimension `[k, 3]`.
+encoded as a tensor of dimension `[k, 3]`, where the first `k - 1` buckets
```

Comment:
This module docstring is all referring to the legacy "tb.summary" APIs (for TensorFlow 1.x) that were defined in this file, which we aren't changing, so I think it's ok to just leave this docstring as-is (and only have the new format documented in summary_v2.py's docstring).  At some point we should rearrange and clean this up so that's more obvious but probably best to do that for all the summary APIs vs piecemeal.

Reply:
Done.